src/lib/resource.ts

```diff
@@ -434,10 +434,10 @@ export const deviceHardwareList: DeviceHardware[] = [
   },
   {
     hwModel: 72,
-    hwModelSlug: "Seeed_XIAO_S3",
+    hwModelSlug: "Seeed XIAO ESP32-S3",
```

Comment:
Hi Dylan,
Hardware model slug needs to match this macro in architecture.h or it will fail. 
![image](https://github.com/user-attachments/assets/24e2beac-5988-4161-9677-f97124fabc9b)

Display name looks good though